websites/B/Bungie/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "RedBird16.com",
+    "id": "444872133504663552"
+  },
+  "service": "Bungie",
+  "description": {
+    "en": "Bungie.net is the home for all your Destiny/Destiny Two needs!"
+  },
+  "url": ["www.bungie.net", "bungie.net", "comics.bungie.net", "bungiestore.com"],
+  "version": "1.0.0",
+"logo": "https://i.imgur.com/7N4tOFk.jpg",
+"thumbnail": "https://imgur.com/a/szETlTp",
```

Comment:
Link should be an image, not an album

Reply:
Yep, you're right. I forgot about that. I'm sorry!